src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/kafka/KafkaSyketilfellebit.kt

```diff
@@ -15,3 +16,10 @@ data class KafkaSyketilfellebit(
     val tom: LocalDate,
     val korrigererSendtSoknad: String?,
 )
+
+fun KafkaSyketilfellebit.isRelevantForOppfolgingstilfelle(): Boolean = isArbeidstakerBit() || isNotArbeidstakerBit()
+
+private fun KafkaSyketilfellebit.isArbeidstakerBit(): Boolean = this.orgnummer != null
+
+private fun KafkaSyketilfellebit.isNotArbeidstakerBit(): Boolean =
```

Comment:
Går det an å si hva disse tilfellene er (i stedet for hva de **ikke** er)?

Reply:
Det går ann, isBekreftetSykmelding er jo en mulighet, men skjønner man hva det er?

Det jeg prøver å beskrive her en en bit for en person som er arbeidsledig, permittert, frilanser, selvstendig næringsdrivene eller annet(hvor fellesnevner er at det ikke er snakk om en arbeidstaker) i en slags sekkebetegnelse.

Reply:
Det som er mest forvirrende for meg, er at det høres ut som at disse burde inneholde hele settet med biter:
```
isNotArbeidstakerBit(): Boolean = !isArbeidstakerBit()
```
Men det ser ikke ut til å stemme? 🤔 Er ikke alle sykmeldinger som er "bekreftet" per definisjon for en ikke-arbeidstaker?

Reply:
Jo, det stemmer det. Er bare litt kjipt å måtte spørre team sykmelding hva en bekreftet sykmelding egentlig er hver gang man lurer på det. Men det kan jo evt. løses med info i README eller en kommentar.

Reply:
Oppdatert til:
```
fun KafkaSyketilfellebit.isRelevantForOppfolgingstilfelle(): Boolean = isArbeidstakerBit() ||
    this.tags.containsAll(listOf(Tag.SYKMELDING.name, Tag.BEKREFTET.name))
```